homepage/templates/faq.html

```diff
@@ -0,0 +1,62 @@
+                    <p>Widget documentation available on our <a href="https://orange.biolab.si/toolbox/">website</a>. We also provide an e-mail course (<a href="https://biolab.us15.list-manage.com/subscribe?u=2cad1f1cbf3a532a77efe4354&id=6f257f15ff">subscribe here</a>), beginner-level <a href="https://www.youtube.com/playlist?list=PLmNPvQr9Tf-ZSDLwOzxpvY-HrE0yv-8Fy">YouTube tutorials</a> and a regularly updated <a href="https://blog.biolab.si/">blog</a>.</p>
+
+                    <h2>I have a question concerning data mining, statistics and machine learning algorithms.</h2>
+                    <p>Visit <a href="https://datascience.stackexchange.com/questions/tagged/orange">Data Science StackExchange</a> and ask the community!</p>
+
+                    <h2>What data does Orange support?</h2>
+                    <p>Core Orange supports Excel, comma- and tab-delimited files (.xlsx, .csv, .tab). It also reads online data, such as Google Spreadsheets. Orange3-ImageAnalytics add-on can import images (.jpg, .tiff, .png). Orange3-Corpus add-on can import text files (.txt, .docx, .odt). SQL widget supports Postgres and MySQL databases.</p>
+
+                    <h2>I want to use Orange on big data.</h2>
+                    <p>Orange provides SQL widget, which samples data instances and enables exploratory data analysis on large datasets. You need <a href="http://initd.org/psycopg/">psycopg2</a> module installed to be able to see the widget. Install it directly in Python with <code>pip install psycopg</code> command. You can also use <a href="https://www.2ndquadrant.com/en/resources/2uda/">2UDA package</a> for easier installation. Please note that Orange supports only Postgres and MySQL databases for now.</p>
+
+                    <h2>Can I export Orange workflow as a Python script?</h2>
+                    <p>Unfortunately, no. We've debated this long and hard and it is simply not possible to maintain interactive analysis and export the workflow as a Python script at the same time. And interactive visualizations and exploratory data analysis are the very heart of Orange.</p>
+
+                    <h2>I wish to propose a collaboration and/or a project with Orange team.</h2>
+                    <p>Please submit a contact request and we will respond as soon as we can.</p>
+
+                    <h2>I wish to develop a widget / an add-on for Orange.</h2>
+                    <p>We are always happy to receive contributions! Development documentation for <a href="http://orange-development.readthedocs.io/">widgets</a> and <a href="https://github.com/biolab/orange3-example-addon">add-ons</a> is available on the web. Next, submit a pull request and our team will review it. If you have specific questions, use <a href="https://gitter.im/biolab/orange3">Gitter</a> for direct communication with the developers.</p>
+
+                    <h2>I wish to commission a custom module and/or an add-on in Orange.</h2>
```

Comment:
in Orange -> for Orange ?

Will everybody understand that the word commission implies paying for it?